openedx/core/djangoapps/kafka_consumer/docs/0001_kafka_consumer_command.rst

```diff
@@ -0,0 +1,39 @@
+Managing Kafka Consumers
+--------------
+
+Status
+======
+
+In Progress
+
+Context
+=======
+As outlined in the upcoming OEP-52, we have elected to go with Apache Kafka as
+our event bus implementation. The standard pattern for consuming events with
+Kafka is to poll in a loop and process messages as they come in. According to
+the Confluent team, it is a best practice to limit each consumer to a single
```

Comment:
You might want to add a note about "Confluent", and why their opinion is relevant.